openshift-provision/openshift-cluster-create.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: openshift-install
+spec:
+  workspaces:
+    - name: install-dir
+      mountPath: /tekton/home/
+    - name: secrets
+  params:
+    - name: PLATFORM
+      type: string
+      description: Platform name where the openshift cluster needs to be deployed such as AWS, GCP etc
+      default: ""
+    - name: PROJECT_ID
+      type: string
+      default: ""
+      description: Project Id required in case of gcp
+    - name: CLUSTER_NAME
+      type: string
+      description: Cluster name
+    - name: BASE_DOMAIN
+      type: string
+      description: Base domain of your cluster install
+    - name: REPLICAS
+      description: How many replicas for the master and worker node
+      type: string
+    - name: REGION
+      type: string
+      description: AWS Region where to install
```

Comment:
Instead of specifying a particular platform can you specify general way
Because it can be any platform like GCP, AWS etc...